src/Core/Settings.cpp

```diff
@@ -59,6 +59,11 @@ namespace ErrorCodes
   *
   * When adding new or changing existing settings add them to the settings changes history in SettingsChangesHistory.cpp
   * for tracking settings changes in different versions and for special `compatibility` settings to work correctly.
+  *
+  * The settings in this list are used to autogenerate the markdown documentation. You can find the script which
+  * generates the markdown from source here: https://github.com/ClickHouse/clickhouse-docs/blob/main/scripts/settings/autogenerate-settings.sh
+  *
+  * If a setting is only available on Cloud, then please include in the description: "Only in ClickHouse Cloud."
```

Comment:
`only available on Cloud` is a misnomer. All settings in this file are technically available in ClickHouse OSS as well (users may set them), they just have no effect. We should write something along the lines `If a setting has an effect only in ClickHouse Cloud, ...`

Similarly: `Only in ClickHouse Cloud.` --> `Only has an effect in ClickHouse Cloud.`.

Reply:
(the same applies to the settings in this file between `read_through_distributed_cache` and `filesystem_cache_enable_background_download_during_fetch`)